src/Standards/BestIt/Sniffs/Commenting/AbstractDocSniff.php

```diff
@@ -392,33 +336,34 @@ public function getRegisteredTokens(): array
      *
      * @return void
      */
-    public function processToken()
+    protected function processToken(): void
     {
         $isVariable = false;
 
         $propertyHelper = new PropertyHelper($this->getFile());
 
-        if ($this->getListenerToken()['code'] === T_VARIABLE
-            && !$propertyHelper->isProperty($this->getListenerPointer())
+        if ($this->getToken()['code'] === T_VARIABLE
+            && !$propertyHelper->isProperty($this->getStackPosition())
```

Comment:
"Pointer" is usually used in a more complicated way and means something else in computer science, so i changed to much simpler "position".